### ISSUE TITLE
tidb-ctl 组件已经不维护，建议从最新文档中移除

### ISSUE THREAD

**Issue by shawn0915** [pingcap/docs-cn#16122]
## Change Request

**This repository is ONLY used to solve problems related to DOCS-CN.
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

master文档中有若干关于 `tidb-ctl` 组件的描述，但该组件已不再维护，类似于 [TiUniManager](TiUniManager), 建议从最新文档中移除。
![图片](https://github.com/pingcap/docs-cn/assets/17957066/6f3c9746-72cf-4ddf-89cd-7fca949fa34b)

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

https://asktug.com/t/topic/1020300

**Comment by rpaik** [comment]
👋 @shawn0915 I removed the Docs Dash label, but this will count towards your TIDB Docs Dash contribution for opening a new issue.

**Comment by qiancai** [comment]
yes, we will count all contribution issues created during the Docs Dash event. 

Here is a list of them: https://github.com/orgs/pingcap/projects/51/views/14
